src/widgets/notesubfoldertree.h

```diff
@@ -21,8 +21,9 @@ class NoteSubFolderTree : public QTreeWidget {
 
     static QMenu *contextMenu(QTreeWidget *parent);
 
-   private Q_SLOTS:
```

Comment:
What's the reason for this change?

Reply:
I mean it not being a slot anymore...

Reply:
I just needed to make it public, and I wasn't sure if it's also required to be a slot, or how to do it.

Reply:
Yeah it has to be public in order to be accessed in the connect() call.

Reply:
Okay, I made them slots in the latest commit.